scripts/lockedgen/main.go

```diff
@@ -20,20 +23,35 @@ import (
 
 func main() {
```

Comment:
Can you add `// +build ignore` for this script?

Reply:
Not sure why?

Reply:
just to avoid having one more executable `lockedgen` on path. Like other scripts in this folder.

Reply:
anyways did the move and added tag